authentication/managing_cloud_provider_credentials/cco-mode-mint.adoc

```diff
@@ -0,0 +1,69 @@
+[id="cco-mode-mint"]
+= Using mint mode
+include::modules/common-attributes.adoc[]
+:context: cco-mode-mint
+toc::[]
+
+Mint mode is supported for Amazon Web Services (AWS), Microsoft Azure, and Google Cloud Platform (GCP).
+
+Mint mode is the default and recommended best practice setting for the Cloud Credential Operator (CCO) to use on the platforms for which it is supported. In this mode, the CCO uses the provided administrator-level cloud credential to create new credentials for components in the cluster with only the specific permissions that are required.
+
+If the credential is not removed after installation, it is stored and used by the CCO to process `CredentialsRequest` CRs for components in the cluster and create new credentials for each with only the specific permissions that are required. The continuous reconciliation of cloud credentials in mint mode allows actions that require additional credentials or permissions, such as upgrading, to proceed.
+
+If the requirement that mint mode stores the administrator-level credential in the cluster `kube-system` namespace does not suit the security requirements of your organization, see _Alternatives to storing administrator-level secrets in the kube-system project_ for xref:../../installing/installing_aws/manually-creating-iam.adoc#alternatives-to-storing-admin-secrets-in-kube-system_manually-creating-iam-aws[AWS], xref:../../installing/installing_azure/manually-creating-iam-azure.adoc#alternatives-to-storing-admin-secrets-in-kube-system_manually-creating-iam-azure[Azure], or xref:../../installing/installing_gcp/manually-creating-iam-gcp.adoc#alternatives-to-storing-admin-secrets-in-kube-system_manually-creating-iam-gcp[GCP].
+
+[id="mint-mode-permissions"]
+== Mint mode permissions requirements
+When using the CCO in mint mode, ensure that the credential you provide meets the requirements of the cloud on which you are running or installing {product-title}. If the provided credentials are not sufficient for mint mode, the CCO cannot create an IAM user.
```

Comment:
Is this done automatically? As suggested in _[Default behavior](https://github.com/openshift/openshift-docs/pull/29545/files#diff-76786602ed2bd71315fdf4b992f73b242cf54a482cf18d893838489034b7405eR75)_?

Reply:
Tracking in [OSDOCS-1817](https://issues.redhat.com/browse/OSDOCS-1817)